class.jetpack-jitm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Jetpack just in time messaging through out the admin
+ *
+ * @since 3.7.0
+ */
+class Jetpack_JITM {
+
+	/**
+	 * @var Jetpack_JITM
+	 **/
+	private static $instance = null;
+
+	static function init() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new Jetpack_JITM;
+		}
+
+		return self::$instance;
+	}
+
+	private function __construct() {
+		$jetpack_hide_jitm = Jetpack_Options::get_option( 'hide_jitm' );
+
+		if ( ! Jetpack::is_module_active( 'photon' ) && 'hide' != $jetpack_hide_jitm['photon'] ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'jitm_enqueue_files' ) );
+			add_action( 'post-upload-ui', array( $this, 'photon_msg' ) );
+		}
+	}
+
+	/*
+	 * Present Photon just in time activation msg
+	 *
+	 */
+	function photon_msg() {
+		if ( current_user_can( 'jetpack_manage_modules' ) ) { ?>
+			<div class="jp-jitm"><a href="#"  data-module="photon" class="dismiss"><span class="genericon genericon-close"></span></a>
+				<p><span class="icon"></span>
+					<?php _e( 'Mirror your images to our free Jetpack CDN to deliver them to your visitors optimized and faster than ever.', 'jetpack' ); ?>
```

Comment:
Are you solid on this wording?  In particular the first half (Mirror your images to our free Jetpack CDN) doesn't resonate with me.  It sounds too technical IMO